lib/hardware_service.py

```diff
@@ -0,0 +1,12 @@
+class HardwareService(object):
```

Comment:
All my files are separate files. You can always delete them in no time.

The code in bin and libquads.py is the Red-Hat code. I think we should use it as a baseline.